cmd/kops/update_federation.go

```diff
@@ -34,7 +34,7 @@ var (
 
 	update_federation_example = templates.Examples(i18n.T(`
 	# After cluster has been edited or upgraded, configure it with:
-	kops update federation k8s-cluster.example.com --yes --state=s3://kops-state-1234 --yes
```

Comment:
Did you mean to keep at least one `--yes`?

Reply:
There is no --yes on the options object unless I missed it